webdriver-spec.html

```diff
@@ -5047,9 +5052,29 @@
    <dd><p>Toggle <var>element</var>’s <a>selectedness</a> state.
 
    <dt><a><code>option</code> element</a>
-   <dd><p>Set <var>element</var>’s <a>selectedness</a> state to true.
+   <dd>
+     <ol>
+       <li><p>Let <var>currently selected</var> is the <a>container</a> with
+         a <a>selectedness</a> state of true.
+
+       <li><p>Let <var>parent node</var> be the calling
+         <a><code>parentNode</code></a> on <var>element</var>'s <a>container</a>.
+
+       <li><p>Run the <a>focusing steps</a> on <var>parent node</var>.
 
-    <p class=issue>Ensure all correct events are fired.
+       <li><p><a>Fire</a> an <a><code>input</code> event</a> at <var>parent node</var>.
+
+       <li><p>Set <var>element</var>’s <a>selectedness</a> state to true.
+
+       <li><p>If <var>currently selected</var> does not equal <var>element</var>:
+         <ol>
+          <li><p><a>Fire</a> a <a><code>change</code> event</a> at <var>parent node</var>.
+         </ol>
+
+       <li><p><a>Fire</a> a <a>mouseUp event</a> at <var>parent node</var>.
+
+       <li><p><a>Fire</a> a <a>click event</a> at <var>parent node</var>.
+     </ol>
```

Comment:
The events here are not entirely in sync with what Marionette does: https://github.com/mozilla/gecko-dev/blob/master/testing/marionette/interaction.js#L201